plotutil/add.go

```diff
@@ -203,8 +204,18 @@ func AddLines(plt *plot.Plot, vs ...interface{}) error {
 				name = ""
 			}
 
+		case *plotter.Function:
+			t.Color = Color(i)
+			t.Dashes = Dashes(i)
+			i++
+			ps = append(ps, t)
+			if name != "" {
+				items = append(items, item{name: name, value: t})
+				name = ""
+			}
+
 		default:
-			panic(fmt.Sprintf("AddLines handles strings and plotter.XYers, got %T", t))
+			panic(fmt.Sprintf("AddLines handles strings, plotter.XYers and *plotter.Function, got %T", t))
```

Comment:
we usually try to have a panic string of the form `"[package name]: [error message]"`
could you change this along these lines?